src/Test/Utilities/Portable/TestUtilities.csproj

```diff
@@ -9,7 +9,7 @@
     <AssemblyName>Roslyn.Test.Utilities</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Nonshipping>true</Nonshipping>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFrameworks>netstandard1.3;$(RoslynPortableTargetFrameworks)</TargetFrameworks>
```

Comment:
That only works if we have conditional `ProjectReference` items. Tests like CSharpSymbols would then need to reference `TestUtilities.CoreClr` only when building for netcoreapp. That isn't supported today and seems a bit crazy.